src/compiler.js

```diff
@@ -831,7 +831,8 @@ var Compiler = Object.extend({
             '[' + argNames.join(', ') + '], ',
             '[' + kwargNames.join(', ') + '], ',
             'function (' + realNames.join(', ') + ') {',
-            'frame = frame.push(true);',
+            'callerFrame = frame;',
```

Comment:
Should this be `var callerFrame = frame;`? Global variables may cause potential bugs.